nautobot/ipam/models.py

```diff
@@ -1293,6 +1293,20 @@ class VLANGroup(OrganizationalModel):
     )
     description = models.CharField(max_length=CHARFIELD_MAX_LENGTH, blank=True)
 
+    range = JSONArrayField(
+        base_field=models.PositiveIntegerField(
+            validators=[
+                MinValueValidator(constants.VLAN_VID_MIN),
+                MaxValueValidator(constants.VLAN_VID_MAX),
+            ]
+        ),
+        verbose_name="Allowed vlan identifiers",
```

Comment:
So in the default case, this will store the default range 1-4094 as an expanded JSON array, i.e. `[1, 2, 3, 4, 5, ..., 4092, 4093, 4094]`?  How impactful is that on storage size and performance? Wonder if it might be preferable or even necessary to store the collapsed range as a list of `(start, end)` tuples instead of fully expanding it out into a list in the DB?

Reply:
I did some measurements and they are as follows

```
nautobot=# SELECT name, pg_column_size(range) as rowsize FROM ipam_vlangroup;
    name    | rowsize
------------+---------
 test1      |      17
 test1-10   |     125
 test1-100  |    1208
 test1-1000 |    3410
 test1-4094 |   13873
(5 rows)
```

Each 1 "megabyte" would store 72 vlan groups with full range of 1-4094.

While this is not optimal with regards storage, we could keep the code cleaner and keep this contribution simple. On the implementation side of things, I also don't see this model used a lot.

Reply:
Hmm, so even `1-100` is already over 1k size... I'd be concerned about query performance issues with Postgres TOAST (a la https://hakibenita.com/sql-medium-text-performance), and especially with the default case being `1-4094`.

Reply:
Yeah I have to agree, I'm nut sure I understand why we need to expand the array for storage? Especially when the range containment math, and even range optimization, is relatively trivial.